.github/workflows/first_interaction.yml

```diff
@@ -0,0 +1,16 @@
+name: First interaction
+
+on:
+  pull_request:
+  issues:
+
+jobs:
+    name: Run
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: TylerLeonhardt/first-interaction-pwsh@master
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: "#### Hi 👋\nCongrats on opening your first issue!\nIf you are looking for more information on how to contribute check out our [contribution guide](https://github.com/ory/meta/blob/master/CONTRIBUTING.md)."
```

Comment:
There is an emoji after `Hi`?

Reply:
yea like a waving hand.

Do you think we can insert a random emoji here from a list?

Reply:
Make a PR against the action to insert a string into the string at a given position, e.g. using some template engine.
This way you can also insert the name of the person.

Reply:
Then if you provide a list of strings it can randomly select one.

Reply:
Btw, you can have multiline strings in yaml, this would be much more readable.